### ISSUE TITLE
How to save model when training?

### ISSUE THREAD

**Issue by pablovela5620** [open-mmlab/mmpose#92]
So I'm trying to learn how to use this repo, but I can't figure out how to save the model. I assumed it would be saved in the same directory as the logs are created but don't see any .pth files there.

I've done the following when training a resnet50 model on the mpii dataset on 2 gpus
```
./tools/dist_train.sh configs/top_down/resnet/mpii/res50_mpii_256x256_test.py 2
```
I have a pretrained model at `models/pytorch/imagenet/resnet50-19c8e357.pth`

I train for 2 epochs and get the .log and .log.json files + a pred.mat though I'm not sure what the .mat format is?
Where does the .pth file save? I've tried digging into the code but haven't really found where and it seems like currently, I can only finetune to save a model? does it only save based on a certain eval metric (PCKh/mAP)?

Any help would be appreciated, thanks!

**Comment by jin-s13** [comment]
Hi，
In the config file，you can find `checkpoint_config = dict(interval=10)`.
which means that we will save the model every 10 epochs. 
Please change this line if needed.

**Comment by jin-s13** [comment]
By the way, the pred.mat file (in Matlab format) contains the prediction results. Since the official MPII dataset prefers the Matlab format, so we also follow this common setting and provide the results in .mat

**Comment by pablovela5620** [comment]
Exactly what I was looking for, thank you for the help! You guys are super fast
